node/service.json

```diff
@@ -15,6 +15,14 @@
       "path": "/sitemap-brands.xml",
       "public": true
     },
+    "custom": {
+      "path": "/sitemap-custom.xml",
+      "public": true
+    },
+    "user": {
+      "path": "/sitemap-user.xml",
```

Comment:
why not being more specific and change . this route to "/sitemap-user-routes.xml"